.github/ISSUE_TEMPLATE/ask_question.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Ask a question
+about: Asking a question related gRPC-Java
+labels: question
+---
+
+Please answer these questions before submitting a bug report.
```

Comment:
maybe "Please read these guidelines before submitting a question."?

Reply:
good catch. actually i removed the line because there is no question in this template.